src/pytesmo/__init__.py

```diff
@@ -19,4 +44,4 @@
 
 testdata_path = os.path.join(src_path, '..', 'tests', 'test-data')
```

Comment:
I wonder if there is a better solution, in the tests it is nice to import the testdata path... but maybe this should not be here...

Reply:
I think there is a better solution, but I have to look it up

Reply:
testdata_path is only used in the examples anyways, I changed the examples accordingly and removed it from the package.